src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -147,9 +183,15 @@ define(function (require, exports, module) {
                 documentStore = this.getFlux().store("document"),
                 currentDocument = documentStore.getDocument(documentID),
                 currentLayer = currentDocument.layers.byID(this.props.layer.id);
-
-            this.getFlux().actions.groups.setGroupExpansion(currentDocument, currentLayer,
-                !currentLayer.expanded, descendants);
+            
+            // Performance Hack: delay the expand action to promote browser paint event.
+            Promise
+                .bind(this)
+                .delay(10)
```

Comment:
I'm curious about how this constant affects things. Does it work the same way with `.delay(0)`? How about `setTimeout(fn, 0)`? In theory, bluebird should now be treating `delay(0)` like `process.nextTick` (i.e., executing asynchronously but _in the same frame_) whereas `delay(n)` for `n > 0` it should be treating like `setTimeout(fn, n)` (i.e., asynchronously and _in a new frame_).